src/tests/ftest/pool/create_all.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+"""
+  (C) Copyright 2018-2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from pool_test_base import PoolTestBase
+import json
+
+
+class PoolCreateAllTest(PoolTestBase):
+    # pylint: disable=too-few-public-methods
+    """Tests pool create all basics
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a SimpleCreatePoll object."""
+        super().__init__(*args, **kwargs)
+
+    def setUp(self):
+        """Set up the test case."""
+        super().setUp()
+        self.update_available_bytes()
+
+    def update_available_bytes(self):
+        """Update the available size of the tiers storage."""
+        self.log.info("Retrieving available size")
+
+        self.scm_bytes = 0
+        self.smd_bytes = 0
+
+        self.assertGreater(len(self.server_managers), 0, "No server managers")
+
+        try:
+            result = self.server_managers[0].dmg.storage_query_usage()
+        except CommandFailure as error:
+            self.fail("dmg command failed: {}".format(error))
+
+        for host_storage in result["response"]["HostStorage"].values():
+            for scm_devices in host_storage["storage"]["scm_namespaces"]:
+                self.scm_bytes += scm_devices["mount"]["avail_bytes"]
+            for nvme_device in host_storage["storage"]["nvme_devices"]:
+                for smd_device in nvme_device["smd_devices"]:
+                    self.smd_bytes += smd_device["avail_bytes"]
+
+        self.log.info(f"Available Bytes: scm={self.scm_bytes}, smd={self.smd_bytes}")
```

Comment:
(pylint-logging-fstring-interpolation) Use lazy % formatting in logging functions